packages/core/src/sass/app/_common/layout/header.scss

```diff
@@ -33,6 +33,52 @@
         padding-bottom: 1.6rem;
         margin-bottom: 1.6rem;
     }
+
+    &__-mobile-theme--tradinghub {
```

Comment:
can we move this to the above ?
&__menu-mobile-theme {
        border-bottom: 1px solid var(--general-section-1);
        padding-bottom: 1.6rem;
        margin-bottom: 1.6rem;
        &--tradinghub {
            padding-bottom: 1.6rem;
            margin-bottom: 1.6rem;
        }
    }